### PR TITLE
Add checkstyle to restrict for more than one line space

### PR DESCRIPTION
**Pull request by dharmendrak**
### Change description ###
Add checkstyle to restrict for more than one line space

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java [hunk 1]
```diff
@@ -323,8 +326,19 @@ public static CCDCase getCCDCitizenCase(List<CCDCollectionElement<CCDAmountRow>>
             .interestClaimStartDate(LocalDate.now())
             .interestSpecificDailyAmount(BigDecimal.valueOf(10))
             .interestEndDateType(CCDInterestEndDateType.SUBMISSION)
+            .paymentStatus("success")
```
Comment (srineelesh) [inline ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java]: this does not seem related to the PR and is not just a formatting change. This change should have been part of a separate PR.

File: domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java [hunk 2]
```diff
@@ -90,6 +92,20 @@ public static Claim getDefault() {
             ).build();
     }
 
+    public static Claim withFullClaimData() {
+        return builder()
+            .withClaimData(SampleClaimData.builder()
+                .withExternalId(RAND_UUID)
```
Comment (srineelesh) [inline domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java]: same as above

File: domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SamplePayment.java [hunk 3]
```diff
@@ -14,7 +14,9 @@ private SamplePayment() {
     public static PaymentBuilder builder() {
         return Payment.builder()
             .reference("RC-1524-6488-1670-7520")
-            .amount(new BigDecimal("4000"))
+            .amount(BigDecimal.valueOf(4000))
```
Comment (srineelesh) [inline domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SamplePayment.java]: same as above.. The concern is I can't see or make out easily what these changes relate to

**Comment by KiranVarma02** [comment]
@srineelesh i was wondering about that, not sure how this check-style thing works, i thought it may be something which runs and removes all files where there is more than one line space. 

@dharmendrak could you please clarify...

File: domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SamplePayment.java (hunk 3 above)
Comment (dharmendrak) [reply domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SamplePayment.java]: test coverage

File: domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java (hunk 2 above)
Comment (dharmendrak) [reply domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java]: test coverage

File: ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java (hunk 1 above)
Comment (dharmendrak) [reply ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java]: test coverage

File: config/checkstyle/checkstyle.xml [hunk 4]
```diff
@@ -75,6 +75,9 @@
             <message key="ws.notPreceded"
                      value="WhitespaceAround: ''{0}'' is not preceded with whitespace."/>
         </module>
+        <module name="WhitespaceAfter">
+            <property name="tokens" value="COMMA, SEMI"/>
```
Comment (singh-surjeet) [inline config/checkstyle/checkstyle.xml]: Is that allowing whitespace after comma or semi colon?
